x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/jwt/JwtSignatureValidator.java

```diff
@@ -294,6 +298,11 @@ public void validate(String tokenPrincipal, SignedJWT signedJWT, ActionListener<
                         );
                         listener.onFailure(primaryException);
                         return;
+                    } else {
+                        logger.debug(
+                            "Successful reload of JWK set. Now with sha256=[{}]",
+                            MessageDigests.toHexString(maybeUpdatedContentAndJwksAlgs.sha256())
```

Comment:
To help correlate with previous messages, I think we want to log the original sha256 as well.

Reply:
I fixed that correlation in an a prior PR ... i don't think i had an example in the OP but here is a rotation at debug level:
```
[2023-08-22T17:51:40,210][DEBUG][o.e.x.s.a.j.J.PkcJwtSignatureValidator] [runTask-0] Attempting to reload JWK set with sha256=[600e0fd89b8088eb3f1a7e7e86c295c55f0a183e6a3f21897d1aa2bf828a5cc6]
[2023-08-22T17:51:40,211][DEBUG][o.e.x.s.a.j.JwkSetLoader ] [runTask-0] Usable PKC: JWKs=[2] algorithms=[RS256] sha256=[be32729066861a35d1e9212c82a500ac10111d837bff34dee0b3596084deb0ad]
[2023-08-22T17:51:40,211][DEBUG][o.e.x.s.a.j.JwkSetLoader ] [runTask-0] Reloaded JWK set from sha256=[600e0fd89b8088eb3f1a7e7e86c295c55f0a183e6a3f21897d1aa2bf828a5cc6] to sha256=[be32729066861a35d1e9212c82a500ac10111d837bff34dee0b3596084deb0ad]
[2023-08-22T17:51:40,211][DEBUG][o.e.x.s.a.j.J.PkcJwtSignatureValidator] [runTask-0] Successful reload of JWK set. Now with sha256=[be32729066861a35d1e9212c82a500ac10111d837bff34dee0b3596084deb0ad]
```
It's a little redundant, but they are different parts of the system and if there are concurrent issues the the duplication could help.